src/api/votes.ts

```
@@ -58,3 +58,45 @@ export const createVote = async (voteData: CreateVoteData) => {
+    params.append('sort', sort)
+
+    const response = await authApi.get<MineVotesResponse>(
+      `/votes/mine/created?${params.toString()}`,
+    )
+    return response.data
+  } catch (error) {
+    throw error
+  }
+}
+
+export const fetchMineVotesVoted = async (
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)